pulse_actions/handlers/route_functions.py

```diff
@@ -0,0 +1,18 @@
+import pulse_actions.handlers.treeherder_buildbot as treeherder_buildbot
+import pulse_actions.handlers.treeherder_resultset as treeherder_resultset
+import logging
+
+logging.basicConfig(format='%(levelname)s:\t %(message)s')
+LOG = logging.getLogger()
+
+def route(data, message, dry_run):
+    if 'action' in data:
+        if data["action"] == "backfill":
+            treeherder_buildbot.on_buildbot_event(data, message, dry_run)
+        elif data["action"] in ["trigger_missing_jobs", "trigger_all_talos_jobs"]:
+            treeherder_resultset.on_resultset_action_event(data, message, dry_run)
+        else:
+            # We don't want to take any action on 'retrigger' or 'cancel' events
+            LOG.debug("We currently only support 'backfill', 'trigger_missing_jobs', 'trigger_all_talos_jobs' for multiple listeners.")
```

Comment:
Makes sense. Is there any key that only exists in one exchange and not the other? That way you could detect from which exchange the message is coming from and pass it to the right function without having to look directly at data['action'], and you would only have to change things in one place the next time you add an action. But this way works if there is no convenient key.